src/JoomlaBrowser.php

```diff
@@ -442,28 +442,48 @@ public function uninstallExtension($extensionName)
 		$this->debug('Extension successfully uninstalled');
 	}
 
-	/**
-	 * Function to Search For an Item in Joomla! Administrator Lists views
-	 *
-	 * @param   String  $name  Name of the Item which we need to Search
-	 *
-	 * @return void
-	 */
-	public function searchForItem($name = null)
-	{
-		$I = $this;
-		if($name)
-		{
-			$I->debug("Searching for $name");
-			$I->fillField(['id' => "filter_search"],$name);
-			$I->click(['xpath' => "//button[@type='submit' and @data-original-title='Search']"]);
-		}
-		else
-		{
-			$I->debug('clearing search filter');
-			$I->click(['xpath' => "//button[@type='button' and @data-original-title='Clear']"]);
-		}
-	}
+    /**
+     * Function to Search For an Item in Joomla! Administrator Lists views
+     *
+     * @param   String  $name     Name of the Item which we need to Search. If null it cleans the searchbox
+     * @param   array   $options  Optional Array containing the locators of the search buttons
```

Comment:
I started doing it in that way, but this approach has a drawback.
I'll ping you in the chat.